CHANGES.rst

```diff
@@ -113,6 +113,8 @@ Bug Fixes
 
 - Aperture Photometry plugin no longer allows negative counts conversion factor. [#3154]
 
+- Batch mode aperture photometry now works in Cubeviz. [#3163]
```

Comment:
was this broken in the last release or just broken intermediately on main?  Maybe it doesn't need a changelog entry if it was display units that temporarily broke it?  And if it was broken before, should we try to backport?

Reply:
Hmm, one of the tracebacks I fixed did occur in 3.10.3, the other is from unit conversion. Maybe we backport just the first? I wasn't sure if we were going to bother doing a 3.10.4 release.

Reply:
yeah, I'm not sure we'll end up doing the release either, but its nice to have that branch up to date in case anyone wants to stay behind on 3.10 and get updates.  But if it would require a different solution than the one here, probably isn't worth the extra effort - up to you

Reply:
I don't think anything here is incompatible with 3.10.3, let's go ahead and backport. I'll move the changelog entry and add the label.